packages/botonic-nlp/src/tasks/ner/botonic-ner.ts

```diff
@@ -63,37 +62,37 @@ export class BotonicNer {
     return ner
   }
 
+  loadDataset(path: string): Dataset {
+    const dataset = DatasetLoader.load(path)
+    this.entities = Array.from(
```

Comment:
In case you use this construct more than 2 times (https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) create a util function mergeArrays